tf-modules/archive/bootstrap.tf

```diff
@@ -30,7 +30,10 @@ resource "aws_lambda_function" "custom_bootstrap" {
 }
 
 data "aws_lambda_invocation" "custom_bootstrap" {
-  depends_on    = [aws_lambda_function.custom_bootstrap]
+  depends_on = [
+    aws_lambda_function.custom_bootstrap,
+    null_resource.rsa_keys
```

Comment:
What does this do?

Reply:
In kes, there was code that created an RSA public and private key, then stored it to S3. To handle that in TF, I created a `null_resource` that will create those key files if they don't exist. The bootstrap script depends on those keys existing in S3, so this ensures that they are created before the bootstrap is run.